xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -114,7 +119,26 @@ private String exportHTML(XWikiContext context) throws XWikiException, IOExcepti
             name = context.getDoc().getFullName();
         }
 
-        Collection<DocumentReference> pageList = resolvePagesToExport(request.getParameterValues("pages"), context);
+        String[] checkedPages = request.getParameterValues("pages");
+        String[] uncheckedPages = null;
+        String checkedPagesRequestValue = request.get("checked-pages");
```

Comment:
If this data is supposed to be a list why invent your own syntax instead of using standard HTTP one and simply use getParameterValues like we do for pages ?

Reply:
Because here the values are coming from a POST request: it would involve to create as many input field as I have pages checked/unchecked. 
Here it seems cleaner to me to only have 2 input fields and to parse their value: then I need to do it manually.